pages/common/git-setup.md

```diff
@@ -0,0 +1,13 @@
+# git setup
+
+> Setup a Git repository in a specific directory, add all files and make a commit with the added files.
```

Comment:
How about that?
```suggestion
> Setup a Git repository in a specific directory and commit all files.
```
This would also apply to the two example descriptions.